exchange-client.gemspec

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'openstax/exchange/client/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "openstax-exchange-client"
```

Comment:
Possibly just make this `openstax-exchange`?  The fact that it is a gem makes it fairly clear it isn't the full site.  Also it is debatable if this gem is really the client.  Tutor is the client, this gem is more of an adapter.  Taking "client" out of the dir path for the version might also not be so bad.  We have in fact already registered this gem as `openstax_exchange`.

Also, our convention so far is to use underscores to separate words in a gem name: https://rubygems.org/search?utf8=%E2%9C%93&query=openstax